static/js/components/email/EmailCompositionType.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import React from 'react';
+import { RadioButton, RadioButtonGroup } from 'material-ui/RadioButton';
+
+import {
+  ONE_TIME_EMAIL,
+  EMAIL_CAMPAIGN
+} from './constants';
+
+export default class EmailCompositionType extends React.Component {
+  props: {
+    emailCompositionType:    ?string,
+    setEmailCompositionType: (b: string) => void,
+  };
+
+  handleRadioClick = (value: string): void => {
+    const { setEmailCompositionType } = this.props;
+    setEmailCompositionType(value);
+  }
+
+  renderEmailCampaign = (): React$Element<*> => (
+    <div className="email-campaign-content">
+      This email will be sent now and in the future whenever users meet the criteria.
+    </div>
+  );
+
+  render() {
+    const { emailCompositionType } = this.props;
+    let radioIconStyle = {'marginRight': '8px'};
+    let emailCampaign = null;
+
+    if (emailCompositionType === EMAIL_CAMPAIGN) {
+      emailCampaign = this.renderEmailCampaign();
+    }
+
+    return (
+      <div className="email-type">
+        <RadioButtonGroup
+          className="type-radio-group"
+          name="email-composition-type"
+          valueSelected={emailCompositionType}
+          onChange={(event, value)=> this.handleRadioClick(value)}
```

Comment:
could you make `this.handleRadioClick` that `event, value` as it's arguments? then we can avoid creating this inline function.